src/pineko/evolve.py

```diff
@@ -99,7 +99,12 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
 
     """
     operators_card = copy.deepcopy(default_card)
-    x_grid, _pids, _mur2_grid, muf2_grid = pineappl_grid.axes()
+    # Note that this is enough but once we consider max_al different
+    # from 0, it will be better to use the actual order_mask
+    mock_order_mask = np.array([True for _ord in pineappl_grid.orders()])
```

Comment:
I see that this pattern repeats. If ever, we'll be able save this line https://github.com/NNPDF/pineko/pull/75/files#r1138264088